tslint.json

```diff
@@ -0,0 +1,29 @@
+{
+  "extends": ["tslint-config-prettier", "tslint-plugin-prettier"],
+  "rules": {
+    "ban-types": false,
+    "interface-name": false,
+    "no-bitwise": false,
+    "no-duplicate-imports": true,
+    "no-invalid-template-strings": true,
+    "no-object-literal-type-assertion": true,
+    "no-sparse-arrays": true,
+    "no-var-requires": false,
+    "prefer-conditional-expression": true,
+    "prefer-object-spread": false,
```

Comment:
Let's turn this on

Reply:
https://github.com/wireapp/wire-web-packages/pull/563/commits/437adcede86ce4cc51f01c8d4973086383234bea